assets/json/campsitesfinal.json

```diff
@@ -10159,6 +10167,14 @@
     "coordinates": "37.226611, -121.97468",
     "photoUrl": ""
   },
+  {
+    "url": "https://www.facebook.com/groups/valley.worx.modesto/",
```

Comment:
This URL is not publicly visible, hence we cannot add it. Can you please check and update?

Reply:
Yes that's what I meant u will change it today

Reply:
Okay great.

Reply:
Will merge as soon as we can confirm its public, thanks.

Reply:
@mtpjr88 Hi, just to clarify there are two entries for Modesto, one with url `https://www.facebook.com/groups/free.code.camp.modesto/` which is a working public group, and one with url `https://www.facebook.com/groups/valley.worx.modesto` which is either a bad url or a closed group. I can see you are an admin on the first one so I assume you have just entered the wrong url on your PR. The two entries also have slightly differing co-ordinates.  Could you merge the two entries and retain just the correct information please, assuming of course these are not in fact two separate but nearby groups.  thanks